adafruit_displayio_layout/widgets/dial.py

```diff
@@ -135,6 +136,22 @@ class Dial(Widget):
 
        Diagram showing the various parameters for setting the dial labels
        and major and minor tick marks.
+
+    .. figure:: dial_variables_clip_needle.png
```

Comment:
I'm not seeing these images in my docs page when I try to build it locally.  Do these show up for you? I could be doing something wrong, I don't think I've ever built with images before.

Reply:
Yes, these do show up on my builds locally.  Can you see the graphic for the widget (see clip below)?  The widget graphic definitely made it ok onto the official released ReadTheDocs.  I'm a real novice at Sphinx and ReadTheDocs, so maybe you can trace that better than I ever could.

<img width="728" alt="image" src="https://user-images.githubusercontent.com/33587466/111087716-218ff780-84f1-11eb-827d-581775e54f6a.png">

Reply:
Ah, yeah I see that in the published docs. But not my local ones. I must be missing something locally.

Reply:
Here’s a comment about the directory structure that is expected to include figures (absolute vs relative). If it builds but does not show the graphics maybe it’s not finding the image files?

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#images